config/routes.rb

```diff
@@ -710,19 +710,19 @@
 
       resources :notes, only: %i[index show new create], as: :application_choice_notes
 
-      resources :interviews, only: %i[new edit index], as: :application_choice_interviews do
+      resources :interviews, only: %i[new create update edit index], as: :application_choice_interviews do
         collection do
-          get '/new/check', to: 'interviews#check'
-          post '/new/check', to: 'interviews#check'
-          post '/confirm', to: 'interviews#commit'
+          post :preview
```

Comment:
just for consistency with other wizards, maybe we can stay with `commit`?

Reply:
As mentioned earlier, I substituded :commit with :create. `post :preview` is a substitute for `post :check` to avoid having the same controller action handling both the processing/validation of the interview and the rendering of the check page. However I can see how that can cause confusion. In offers we do specifically use a nested `ChecksController` controller and I think that may be the way forward so I plan to investigate that further.

I think we should adop the pattern of using `create` instead of `confirm/commit` however as that is consistent with how rails behaves plus we already use `update` (and not confirm/update) in order to update existing interviews.